src/plugins/ajs-destination/index.ts

```diff
@@ -305,11 +305,11 @@ export class LegacyDestination implements Plugin {
   }
 }
 
-export async function ajsDestinations(
+export function ajsDestinations(
```

Comment:
unless we have a policy that all these plugins need to be async, this looked safe.